iphone/Classes/WebFont.m

```diff
@@ -151,9 +151,9 @@ -(UIFont *) font
                 }
             }
             if (font == nil) {
-                //NO valid family specified. Just check for characteristics. Semi bold is ignored here.
+                //NO valid family specified. Just check for characteristics.
                 if (self.isBoldWeight) {
-                    UIFont* theFont = [UIFont boldSystemFontOfSize:self.size];
+                    UIFont* theFont = [UIFont systemFontOfSize:self.size weight:UIFontWeightBold];
```

Comment:
Actually, we do 8.0 I guess. Maybe we can find out the raw-value and pass that one instead - for backwards-compatibility?